unstructured/ingest/cli/cmds/weaviate.py

```diff
@@ -0,0 +1,70 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig, Dict
+from unstructured.ingest.connector.weaviate import SimpleWeaviateConfig, WeaviateWriteConfig
+
+CMD_NAME = "weaviate"
+
+
+@dataclass
+class WeaviateCliConfig(SimpleWeaviateConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--host-url"],
+                required=True,
+                help="Weaviate instance url",
+            ),
+            click.Option(
+                ["--class-name"],
+                default=None,
+                type=str,
+                help="Name of the class to push the records into, e.g: Pdf-elements",
+            ),
+            click.Option(
+                ["--auth-keys"],
```

Comment:
Reading the docs, this really only needs to support an `api_key` to generate the config: 
```python
auth_config = weaviate.AuthApiKey(api_key="YOUR-WEAVIATE-API-KEY")  # Replace w/ your Weaviate instance API key
```
Let's make it a bit easier to pass in by setting this directly on the cli:
```suggestion
                ["--api-key"],
                type=str
```